src/visualization/Turi Create Visualization/src/js/lib/elements/Plot/Plot.js

```diff
@@ -0,0 +1,166 @@
+import React, { Component } from 'react';
+import vegaEmbed from 'vega-embed';
+import {vega as vegaTooltip} from'vega-tooltip';
+import './Plot.css';
+
+var vega = require('vega');
+
+window.vega_type = vega;
+
+class TcPlot extends Component {
+
+    componentDidMount(){
+        this.addSpec(this.props.vega_spec);
+    }
+
+    checkLoadedFlag(data, $this) {
+        if($this.vegaLoading) {
+            window.setTimeout($this.checkLoadedFlag, 100, data, $this);
+        } else {
+            if($this.vegaLoading){
+                debugger;
```

Comment:
Remove these `debugger` calls? They may get annoying when debugging unrelated issues (if they are supposed to happen frequently).